code-challenges.js

```diff
@@ -16,41 +34,183 @@ const fruit2 = "banana"
+    // undefined
+    // 350 is above boiling point
+    // 212 is at boiling point
+    // learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % 
+
+    // yup getting an undefined for some reason... lol; finally googled it " randomly getting undefined in my function outputs javascript" - every if else needs a return statement and I didnt have one for a few of the iterations. fixed it and now: 
 
+    //learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % node code-challenges.js
+    // banana
+    // kiwi
+    // 42 is below boiling point
+    // 350 is above boiling point
+    // 212 is at boiling point
+    // learnacademy@LEARNs-MacBook-Air-5 week-1-assessment-necessities % 
+
+
+    console.log("--- problem 2 ---")
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
-// Pseudo code:
+    // Pseudo code:
+    // I'll declare an function that takes two inputs. the function needs to combine the two arrays and return the total length. I'll do that by useing the .concat() method to combine them then I will return the the value with the .length method.
 
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+const arrayCombine = (array1, array2) => {
+   
+    return  array1.concat(array2).length
+
+}
+console.log(arrayCombine(myNumbers1, myNumbers2))
 
+    // I actually got 5 an an output, here is my code: 
+    // const arrayCombine = (array1, array2) => {
+    //     array1.concat(array2)
+    //     return array1.length
+
+    // }
+    // console.log(arrayCombine(myNumbers1, myNumbers2))
+    // I'm going to change up the code and combine the methods all onto that array1 and see if it works
+
+    // Yes; once i created this  array1.concat(array2).length on like 117 it now returns 10.
+
+    console.log("--- problem 3 ---")
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
 
-// Pseudo code:
+// Pseudo code: I will declare a function named reverseString that will take one string input and reverse it then output the value. I may take the string and assign each element to a array then reverse it then 
+
+// actually i might do a for loop and interate through the string assigning the values to an array then reverse it
 
 const currentCohort = "Delta 2022"
+    // var reverseSt = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ,11]
+
+    // const reverseString = (inputA) => {
+    // for (i = 0; i < inputA.length; i++) {
+    //     reverseSt.unshift[inputA[""]]
+        
+    // }
+    // console.log(reverseSt)
+    // }
+    // reverseString(currentCohort)
+
+    // my code was not working so I started looking at my notes
+
+//console.log(currentCohort.split(""))
+
+reverseStr = currentCohort.split("")
+
+ // I had to google how to reverse it but the code works. 
+
+console.log(reverseStr.reverse())
```

Comment:
Awesome work, and I really love the breakdown.  Just a note, currently this will return an array and not a string.  You may want to use one more built-in method in the future to return a reversed string.